redbot/cogs/economy/economy.py

```diff
@@ -252,6 +252,27 @@ def __init__(self, bot: Red):
                         credits_name
                     )
                 )
+                await ctx.send(
+                    _("You currently have {} {}.").format(
+                        str((await bank.get_balance(author))), credits_name
+                    )
+                )
+                if await bank.is_global():
```

Comment:
You're doing a `bank.is_global()` check inside of a `bank.is_global()` check here.